src/leveldb/util/bloom.cc

```diff
@@ -47,7 +47,7 @@ class BloomFilterPolicy : public FilterPolicy {
     dst->resize(init_size + bytes, 0);
     dst->push_back(static_cast<char>(k_));  // Remember # of probes in filter
     char* array = &(*dst)[init_size];
-    for (size_t i = 0; i < n; i++) {
+    for (int i = 0; i < n; i++) {
```

Comment:
At least this change is already present in the upstream: https://github.com/google/leveldb/blob/master/util/bloom.cc#L50